service-front/module/Application/view/application/pages/identity_check_passed.twig

```diff
@@ -5,110 +5,98 @@
-                                            class="govuk-label govuk-radios__label"
-                                            for="conditional-assistance-2"
-                                    >
-                                        Yes, give details
-                                    </label>
-                                </div>
-                                <div
-                                        class="govuk-radios__conditional govuk-radios__conditional--hidden"
-                                        id="conditional-assistance"
-                                >
-                                    <div class="govuk-form-group">
-                                    <textarea
-                                            class="govuk-textarea"
-                                            rows="2"
-                                            id="details"
-                                            name="details"
-                                            spellcheck="false"
-                                            aria-label="Details of the assistance provided"
-                                    ></textarea>
-                                    </div>
-                                </div>
-                            </div>
-                        </fieldset>
-                    </div>
-                </fieldset>
+    <div class="govuk-!-width-two-thirds">
+        <div class="moj-banner moj-banner--success" role="region" aria-label="Success">
+            <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false"
+                xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25">
+                <path d="M25,6.2L8.7,23.2L0,14.1l4-4.2l4.7,4.9L21,2L25,6.2z"/>
+            </svg>
+            <div class="moj-banner__message">
+                {% block title %}
+                    Identity check passed.
+                {% endblock %}
             </div>
+        </div>
 
-            <p class="govuk-body">
-            <h2 class="govuk-heading-l">
-                What happens next?
-            </h2>
-            </p>
-            <p class="govuk-body">
-                The next step is we will check your LPA and be in touch with you if we have any further questions. After
-                this, we will be in contact with the certificate provider for the next steps.
-            </p>
+        <h2 class="govuk-heading-m">
+            What happens next?
+        </h2>
 
-            <p class="govuk-body">
+        <div class="govuk-summary-card">
+            <div class="govuk-summary-card__title-wrapper">
+                <h2 class="govuk-summary-card__title">
+                    Read the following to the caller:
+                </h2>
+            </div>
+            <div class="govuk-summary-card__content">
+                <p class="govuk-body">
+                    If  you haven’t  already, please return your paper LPA by post to OPG.
```

Comment:
Need to close the `<p>` tag:
```suggestion
                <p class="govuk-body">
                    If  you haven’t  already, please return your paper LPA by post to OPG.
                </p>
```

Reply:
Well, [technically you don't _need_ to](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p#technical_summary) but gone are the heady 2010s days of intentionally using that quirk.